content/tracing/languages/php.md

```diff
@@ -22,104 +22,254 @@ The APM tracer for PHP applications is in Open Public Beta.
 
 ## Installation and Getting Started
 
-To begin tracing applications written in PHP, first [install and configure the Datadog Agent][1], see additional documentation for [tracing Docker applications][2] or [Kubernetes applications][3].
+For descriptions of terminology used in APM, take a look at the [official documentation][1].
 
-Next, install the Datadog PHP extension using one of the precompiled packages for supported distributions. The latest packages can be found on GitHub's [releases page][4]. If you don't find your distribution, you can install the PHP extension [from source][5]
+For details about contributing the PHP tracer, check out the [contributing guide][2].
+
+### Setup the Datadog Agent
+
+The PHP APM tracer sends trace data through the Datadog Agent.
+
+[Install and configure the Datadog Agent][3], see the additional documentation for [tracing Docker applications][4] or [Kubernetes applications][5].
+
+Make sure the Agent has **[APM enabled][6]**.
+
+### Install the extension
+
+Install the PHP extension using one of the [precompiled packages for supported distributions][7]. If you don't find your distribution, you can install the PHP extension [from PECL][8] or [from source][9].
+
+Once downloaded, install the package with one of the commands below.
 
 ```bash
 # using RPM package (RHEL/Centos 6+, Fedora 20+)
-rpm -ivh datadog-php-tracer.rpm
+$ rpm -ivh datadog-php-tracer.rpm
 
 # using DEB package (Debian Jessie+ , Ubuntu 14.04+)
-deb -i datadog-php-tracer.deb
+$ deb -i datadog-php-tracer.deb
 
 # using APK package (Alpine)
-apk add datadog-php-tracer.apk --allow-untrusted
+$ apk add datadog-php-tracer.apk --allow-untrusted
 
 # using tar.gz archive (Other distributions using libc6)
-tar -xf datadog-php-tracer.tar.gz -C /
-/opt/datadog-php/bin/post-install.sh
+$ tar -xf datadog-php-tracer.tar.gz -C /
+  /opt/datadog-php/bin/post-install.sh
+```
+
+### Install from PECL
+
+<div class="alert alert-warning">
+Installing the PHP tracer via PECL is an experimental feature.
+</div>
+
+Alternatively install the extension from the [PECL package **datadog_trace**][10].
+
+```bash
+$ sudo pecl install datadog_trace-beta
+```
+
+Next, [enable the extension][11].
```

Comment:
We should explain that this step is optional, as per auto-instrumentation :)

Reply:
I should probably change the wording of this a bit since this is a required step when installing the extension via PECL or compiling from source since those methods don't automatically add a  `ddtrace.ini` file anywhere to enable the extension. I'll reword this a bit. :)